packages/grafana-ui/src/components/FormGroup/FormGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { SFC } from 'react';
+import { Label } from '..';
+
+export interface Props {
+  label: string;
+  inputProps: {};
+  labelWidth?: number;
+  inputWidth?: number;
+}
+
+const defaultProps = {
+  labelWidth: 6,
+  inputProps: {},
+  inputWidth: 12,
+};
+
+const FormGroup: SFC<Props> = ({ label, labelWidth, inputProps, inputWidth }) => {
+  return (
+    <div className="gf-form">
+      <Label width={labelWidth}>{label}</Label>
```

Comment:
IMO `label` should be optional and rendered only if present. I can imagine inputs without label, but with placeholder instead

Reply:
To escalate this discussion a bit. I've done and seen too many examples where options grow to jquery ui levels. Imo having more components is a nicer way to go. In this case it could be `<FormFieldWithLabel />` (but probably with a better name) and `<FormField />`. 

I'd like to keep the number of options/optionals to a minimum.